index.bs

```diff
@@ -1431,6 +1431,12 @@ Today writing to {{Document/cookie|document.cookie}} in contexts where script-in
 
 Likewise, today reading {{Document/cookie|document.cookie}} in contexts where script-initiated cookie-reading is disallowed typically returns an empty string. However, a cooperating web server can verify that server-initiated cookie-writing and cookie-reading work and report this to the script (which still sees empty string) and the script can use this information to infer that script-initiated cookie-reading is disallowed.
 
+<!-- ============================================================ -->
+## Clear cookies ## {#clear-cookies}
```

Comment:
Would it make sense to include this in a (new) "Privacy considerations" section?

("Security" should get renamed "Security considerations", and we should eventually add Accessibility and Internationalization considerations sections as well)